src/rastervision/builders/label_store_builder.py

```diff
@@ -3,7 +3,7 @@
 from rastervision.label_stores.classification_geojson_file import (
     ClassificationGeoJSONFile)
 from rastervision.label_stores.segmentation_raster_file import (
-    SegmentationRasterFile)
+    SegmentationInputRasterFile, SegmentationOutputRasterFile)
```

Comment:
Interesting that you decided to split these into two classes. I think it's the case that we either read *or* write to a raster, but never both so I guess it makes sense to isolate the functionality that way. Maybe we should consider doing that (as a later refactor) to the other label stores.

Reply:
:+1: